2-summarizing-data/02-summarizing-numerical-data/notes.qmd

```diff
@@ -291,7 +291,7 @@ $$8, 11, 7, 7, 8, 11,  9,  6,  10,  7,  9$$
 
 The mean, the median, and the mode are the three standard statistics used to measure the center of a distribution. Despite their ubiquity, these three are not carved somewhere on a stone tablet. They're common because they're useful and they're useful because they were constructed very thoughtfully.
```

Comment:
I don't know if we should leave in the reference to Moses and the 10 commandments (or as my partner guessed, the reference to the Rosetta stone). This paragraph will read just fine without it.